console_backend/src/tabs/solution_tab/solution_position_tab.rs

```diff
@@ -424,6 +430,8 @@ impl SolutionPositionTab {
     pub fn handle_pos_llh(&mut self, msg: PosLLH) {
         self.last_pos_mode = msg.mode();
         let pos_llh_fields = msg.fields();
+        self.v_acc = pos_llh_fields.v_accuracy;
```

Comment:
is there a specific orientation / angle for this ellipse?

Reply:
Suggest asking on [#technical-questions](https://snav.slack.com/archives/C02USAD03) -- @ljbade do you know?

Reply:
When there is only a horizontal accuracy then the ellipse should be round. Worth noting horizontal means parallel to the ground plane, vertical means perpendicular to the ground plane.

Thus only horizontal accuracy would be visible in a top down map view.